README.md

```diff
@@ -2,6 +2,13 @@
 
 Simple Migrations for Zend Framework 2. Project originally based on [ZendDbMigrations](https://github.com/vadim-knyzev/ZendDbMigrations) but module author did not response for issues and pull-requests so fork became independent project.
 
+## Added Zend 3 Support 28.01.2022
```

Comment:
Please extract this to the new `CHANGELOG.md` under the `## Unreleased` - will tag it with the version as soon as it will be ready for the release